### DISCUSSION TITLE
`npm version` without commit

### DISCUSSION THREAD

**Discussion by RecuencoJones** [npm/feedback#390] [CLI]
I usually find myself updating `package.json` version manually and then running `npm install` to sync that with `package-lock.json`. This is not ideal as it can potentially update lockfile with unwanted changes other than version.

`npm version` does the same exact job. However, forcing `npm version` to commit seems rather undesired for us.

So, was wondering whether we could add some flag to purposely update versions with it without committing, something in the lines of:

```
npm version minor --no-commit
npm version minor --commit false
npm version minor --no-git
npm version minor --skip-git
npm version minor --unstaged
```

**Comment by ljharb** [comment]
The `git-tag-version` config can be set to `false`, either with `npm config set git-tag-version false` or `NPM_CONFIG_GIT_TAG_VERSION=false npm version whatever`.

**Reply by RecuencoJones** [reply]
Ah, so that was it! I think I completely missread the documentation 🤦 
Thanks a lot!

**Reply by norcino** [reply]
@ljharb sorry I am not familiar with npm/node and I tried to search for it with no success.
Is NPM_CONFIG_GIT_TAG_VERSION an environment variable or something I should set in package.json or maybe a .npmrc file?

**Reply by ljharb** [reply]
it's an environment variable, or `git-tag-version` in `.npmrc`.
